campaignion_vue/campaignion_vue.module

```diff
@@ -58,13 +60,20 @@ function _campaignion_vue_get_strings($language = NULL) {
 function campaignion_vue_preprocess_html(&$vars) {
   $js = drupal_add_js();
   $found = FALSE;
+  $add_dependencies = FALSE;
   foreach ($js as $path => $options) {
-    if (substr($path, -11) == '.vue.min.js') {
+    if (substr($path, -24) == 'redirects_app.vue.min.js') {
```

Comment:
Ok, talked with Florian. I think it’s better to actually migrate the redirect_app too (and remove the campaignion_vue module) instead of maintaining this in-between state.